test/ci-operator-integration/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+WORKDIR="$( mktemp -d )"
+trap 'rm -rf "${WORKDIR}"' EXIT
+
+readonly TEST_ROOT="$( dirname "${BASH_SOURCE[0]}")"
+readonly EXPECTED="${TEST_ROOT}/expected_files/expected.json"
+readonly EXPECTED_WITH_TEMPLATE="${TEST_ROOT}/expected_files/expected_with_template.json"
+readonly DRY_RUN_JSON="${WORKDIR}/ci-op-dry.json"
+readonly DRY_RUN_WITH_TEMPLATE_JSON="${WORKDIR}/ci-op-template-dry.json"
+
+echo "[INFO] Running ci-operator in dry-mode..."
+if ! ci-operator --dry-run --namespace testns --config ${TEST_ROOT}/config/test-config.yaml  --git-ref=openshift/origin@master > ${DRY_RUN_JSON} 2>&1; then
+  echo "ERROR: ci-operator failed:"
+  exit 1
+fi
+
+
+if ! diff <(sort ${EXPECTED} | grep -vE "^[0-9]{4}/[0-9]{2}/[0-9]{2}|sha256:.*") <(sort ${DRY_RUN_JSON} | grep -vE "^[0-9]{4}/[0-9]{2}/[0-9]{2}|sha256:"); then
```

Comment:
So let's use `diff -I` instead of `grep -v` -- we can specify regexes to ignore in `diff` with `-I` and it would be good to do that

Reply:
What is `sort` doing here? sorting the entire JSON file?

Reply:
It seems so but the diff output is helpful enough to find the difference